icu4c/source/tools/toolutil/pkg_genc.cpp

```diff
@@ -878,7 +878,8 @@ writeObjectCode(
         const char *optMatchArch,
         const char *optFilename,
         char *outFilePath,
-        size_t outFilePathCapacity) {
+        size_t outFilePathCapacity,
+        char optSkipDllExport) {
```

Comment:
Optional: The code below might be nicer (avoid double negatives) if you invert this, for example "withDllExport".

Reply:
Done.